src/main/java/com/bitquest/bitquest/BitQuest.java

```diff
@@ -811,6 +811,8 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
                     } catch (IOException e1) {
                         e1.printStackTrace();
                     }
+                    if (sendAmount >= 20000)
+                    {
                     try {
```

Comment:
I can't run a spigot server. There's a lot of it that failed when I tried to run it.